build-utils/language-java/src/main/java/org/apache/plc4x/language/java/JavaLanguageTemplateHelper.java

```diff
@@ -26,13 +26,17 @@ Licensed to the Apache Software Foundation (ASF) under one
 import org.apache.plc4x.plugins.codegenerator.types.fields.*;
 import org.apache.plc4x.plugins.codegenerator.types.references.*;
 import org.apache.plc4x.plugins.codegenerator.types.terms.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Map;
 import java.util.function.Function;
 
 @SuppressWarnings({"unused", "WeakerAccess"})
 public class JavaLanguageTemplateHelper extends BaseFreemarkerLanguageTemplateHelper {
 
+    private final Logger logger = LoggerFactory.getLogger(JavaLanguageTemplateHelper.class);
```

Comment:
I see new logger field but can't see direct use of it.

Reply:
also it should be static and upper case for sonar 😄